apps/sumo-aws-apps/security-hub/securityhub.template.yaml

```diff
@@ -419,15 +419,15 @@ Resources:
     Properties:
       Location:
         ApplicationId: arn:aws:serverlessrepo:us-east-1:956882708938:applications/sumologic-securityhub-forwarder
-        SemanticVersion: 1.0.7
+        SemanticVersion: 1.0.8
```

Comment:
sumologic-securityhub-connector-aws-org 1.0.4
sumologic-securityhub-forwarder 1.0.10
sumologic-securityhub-collector 1.0.9
Please update to latest versions in both the repos.

Reply:
upgraded sumologic-securityhub-forwarder 1.0.10
sumologic-securityhub-collector 1.0.9